### PR TITLE
chore: update frontend type

### PR DESCRIPTION
**Pull request by zxkane**
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "src/**/*.ts"
   ],
   "exclude": [
-    "src/frontend/**",
+    "cdk.out/**/*",
```
Comment (chenhaiyun) [inline tsconfig.json:34]: 下面那行可以不用了吧？

**Review by chenhaiyun — COMMENTED** [review]
666

File: tsconfig.json (hunk 1 above)
Comment (zxkane) [reply tsconfig.json:34]: 曾经build过后，再次执行`npx jest`后执行`cdk.out/`下面的重复用例，所以这里排除了他们

File: tsconfig.eslint.json [hunk 2]
```diff
@@ -26,12 +26,12 @@
...
   "include": [
     ".projenrc.js",
-    "src/**/*.ts",
+    // "src/**/*.ts",
     "test/**/*.ts"
   ],
   "exclude": [
     "node_modules",
-    "src/frontend"
+    "src/frontend/src/**/*"
```
[the remark this answers is not in the document]
Comment (zxkane) [reply tsconfig.eslint.json]: Nice finding. This file will not be generated by `projen`. Removed it.

File: frontend/package.json [hunk 3]
```diff
@@ -6,31 +6,39 @@
     "clobber": "npx projen clobber",
     "compile": "npx projen compile",
     "default": "npx projen default",
-    "dev": "npx projen dev",
```
[the remark this answers is not in the document]
Comment (zxkane) [reply frontend/package.json:9]: added it back

**Review by chenhaiyun — APPROVED** [review]

**Pull request merged**